app/controllers/katello/api/v2/host_collections_controller.rb

```diff
@@ -63,50 +72,55 @@ def create
 
     api :PUT, "/host_collections/:id", N_("Update a host collection")
     param :id, :identifier, :desc => N_("Id of the host collection"), :required => true
-    param :system_uuids, Array, :required => false, :desc => N_("List of content host uuids to be in the host collection")
     param :name, String, :required => false, :desc => N_("Host Collection name")
     param_group :host_collection
     def update
-      @host_collection.update_attributes!(host_collection_params_with_system_uuids)
+      @host_collection.update_attributes!(host_collection_params_with_host_ids)
       respond
     end
 
-    api :PUT, "/host_collections/:id/add_systems", N_("Add content host to the host collection"), :deprecated => true
+    api :GET, "/host_collections/:id/hosts", N_("List hosts in the host collection")
+    param :id, :identifier, :desc => N_("Id of the host collection"), :required => true
+    def hosts
+      hosts = ::Host.authorized(:edit_host).where("#{::Host.table_name}.id" => @host_collection.host_ids)
+      collection = scoped_search(hosts, :name, :desc, :resource_class => ::Host)
+      respond_for_index(:collection => collection, :template => '../hosts/index')
+    end
```

Comment:
so we had started a pattern where we should use the /hosts controller for this and pass ?host_collection_id=X

but we aren't going to easily be able to do this for host_unification.  We can however use the search for that.  Any thoughts about using that existing route and controller?